airbyte-integrations/connectors/source-amazon-seller-partner/source_amazon_seller_partner/streams.py

```diff
@@ -288,8 +289,7 @@ def parse_response(self, response: requests.Response) -> Iterable[Mapping]:
         document_records = self.parse_document(document)
         yield from document_records
 
-    @staticmethod
-    def parse_document(document):
```

Comment:
Hey, isn't this method just as a util?

Reply:
I made it an instance method because in `BrandAnalyticsStream` it accesses instance variable. See https://github.com/airbytehq/airbyte/blob/a97a4ce1b6594be79a9e3bc1a474f99e08bfdad3/airbyte-integrations/connectors/source-amazon-seller-partner/source_amazon_seller_partner/streams.py#L392